src/createRoom.js

```diff
@@ -76,10 +76,7 @@ function createRoom(opts) {
         }
     ];
 
-    let modal;
-    setTimeout(()=>{
-        modal = Modal.createDialog(Loader, null, 'mx_Dialog_spinner');
-    }, 0);
+    const modal = Modal.createDialog(Loader, null, 'mx_Dialog_spinner');
```

Comment:
I don't think there's any way that `modal` can be falsey here, so the `if` on line 83 is redundant now